spotipy/client.py

```diff
@@ -730,7 +715,6 @@ def user_playlist(self, user, playlist_id=None, fields=None, market=None):
 
     def user_playlist_tracks(
         self,
-        user=None,
```

Comment:
to maintain backward-compatibility you can't change the signature of the methods. These methods are deprecated anyway (see `DeprecationWarning` just below) and have new up-to-date methods.